libutils/sequence.c

```diff
@@ -392,3 +392,17 @@ Seq *SeqStringFromString(const char *str, char delimiter)
 
     return seq;
 }
+
+int SeqStringLength(Seq *seq)
+{
+    assert (seq);
+
+    int total_length = 0;
+
+    for (size_t i = 0; i < SeqLength(seq); i++)
```

Comment:
We should avoid calling the ``SeqLength`` function over and over in every iteration of the cycle.

Reply:
Yep